modules/core/shared/src/main/scala/io/circe/Codec.scala

```diff
@@ -55,6 +55,12 @@ trait Codec[A] extends Decoder[A] with Encoder[A] {
 object Codec extends ProductCodecs with ProductTypedCodecs with EnumerationCodecs with CodecDerivationRelaxed {
   def apply[A](implicit instance: Codec[A]): Codec[A] = instance
 
+  /**
+   * Summons a `Codec` where a `Decoder` and an `Encoder` are in implicit scope.
```

Comment:
```suggestion
   * Summons a `Codec` where a `Decoder` and an `Encoder` are in implicit scope.
   * This can cause a runtime infinite loop if you use it and the encoder and decoder is not in implicit scope, and this causes them to become in implicit scope.
```